### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pcm211**
This Is a step by step tutorial to deploy Sharetribe to Amazon AWS Free Tier in Development Mode

### CODE REVIEW

**Comment by bladealslayer** [comment]
@pcm211 I'd suggest rewording the link title to `Deploying Sharetribe to AWS (Free Tier) for development`.

About your instructions in the linked Gist: since this is only your first draft, we'll keep an eye on the instructions and link them once they reach a more finished state. For example, avoiding ALL CAPS and improving the formatting would help readability a lot.

I'd suggest also to work on the wording of the technical terms and describe things more accurately the way they are in AWS. For instance: `CHECK OUTGOING & INCOMING PORTS 22,3000` would be something like `Make sure that your instance's security groups allow incoming connections on TCP ports 22 and 3000`.

**Comment by pcm211** [comment]
done some lipstick work ....  can others edit this too ...to make it better ..??

**Comment by dwidoo** [comment]
I followed these instructions for AWS but after having configured the production section  of `database.yml` with my RDS database details, I get the 2 following issues when I run `bundle exec rake RAILS_ENV=production db:schema:load` :
`You specified server rendering JS file: app/assets/webpack/server-bundle.js, but it cannot be read. You may set the server_bundle_js_file in your configuration to be "" to avoid this warning
rake aborted!`
`Devise.secret_key was not set. Please add the following to your Devise initializer:`

**Comment by rap1ds** [comment]
@dwidoo You need to set the secret key. Check the step one here: https://github.com/sharetribe/sharetribe#setting-up-sharetribe-for-production

**Comment by rap1ds** [comment]
I'm going to close this PR for the reasons @bladealslayer mentioned.
